puppetfile-cli.rb

```diff
@@ -18,6 +18,7 @@ def self.parse(options)
       cache_dir: nil,
       module_paths: [],
       path: nil,
+      proxy: nil,
```

Comment:
Can we add an option for configuring the Forge url here? I can also rebase my PR off this one and add that option if you would like to keep this PR to just the proxy option. :)